builtin/providers/aws/resource_aws_api_gateway_domain_name.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsApiGatewayDomainName() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate_upload_date": &schema.Schema{
```

Comment:
Hey @stack72 
thanks for the review. Not sure what you meant by this... 😞 is it about the `StateFunc`?